src/table/index.js

```diff
@@ -57,36 +57,10 @@ function getFieldValue(cellValue, type, format, dataFormat) {
   }
   return cellValue;
 }
-function isEquivalentObject(a, b) {
```

Comment:
Is this no longer used?

Reply:
Reworked the one function that was using this in the form extras library